tests/features/steps/1_setup.py

```diff
@@ -13,11 +13,18 @@
 class Application(BaseModel):
     app_id: str = "UNSET"
     app_name: str = "UNSET"
-    enable_s3_permissions_lookup: bool = False
-    pointer_types: dict[str, list[str]] = {}
 
-    def add_pointer_types(self, ods_code: str, pointer_types: list[str]):
-        self.pointer_types[ods_code] = pointer_types
+    def add_pointer_types(self, ods_code: str, context: Context):
+        if not context.table:
+            raise ValueError("No permissions table provided")
+
+        pointer_types = [f"{system}|{value}" for system, value in context.table]
+        bucket = f"nhsd-nrlf--{context.env}--authorization-store"
```

Comment:
This bucket name will need changing to `f"nhsd-nrlf--{context.env}-authorization-store"` due to my changes. This is almost like a note to self. It might be a bit more complicated than that as we want to use `dev` instead of `dev-0`